src/test.js

```diff
@@ -64,6 +64,10 @@ export default function Test( settings ) {
 		this.async = false;
 		this.expected = 0;
 	} else {
+		this.module.unskippedTests.push( {
```

Comment:
I think adding the flag is fine, it's an additive change so it should be considered backwards compatible.